curriculum/challenges/espanol/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting/chunky-monkey.md

```diff
@@ -59,7 +59,7 @@ assert.deepEqual(chunkArrayInGroups([0, 1, 2, 3, 4, 5, 6], 3), [
 ]);
 ```
 
-`chunkArrayInGroups([0, 1, 2, 3, 4, 5, 6, 7, 8], 4)` should return `[[0, 1, 2, 3], [4, 5, 6, 7], [8]]`.
+`chunkArrayIngroups([0, 1, 2, 3, 4, 5, 6, 7, 8], 4)` debe devolver `[[0, 1, 2, 3], [4, 5, 6, 7], [8]]`.
```

Comment:
Casing change here should not occur

Reply:
Hmm, that is interesting.  Was this somehow changed via a translation on Crowdin?

Reply:
It was, yes. 

![image](https://user-images.githubusercontent.com/63889819/109872698-a17cae80-7c21-11eb-929a-6eb2b7860931.png)